openshift/pgslice/openshift/build.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: pgslice
+labels:
+  app: pgslice
+  phase: build
+  app.kubernetes.io/name: pgslice
+  app.kubernetes.io/managed-by: template
+parameters:
+  - name: NAME
+    value: pgslice
+  - name: VERSION
+    description: Output version
+    required: true
+    value: "latest"
+  - name: GIT_URL
+    value: https://github.com/bcgov/wps.git
+  - name: GIT_BRANCH
+    value: task/partition-weather-station-model-predictions
```

Comment:
I take it this will be switched to `main` before deployment and merge?

Reply:
Yep